digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/util/DataFixture.scala

```diff
@@ -2056,6 +2060,58 @@ object DataFixture {
     }
   }
 
+  def createRoadWorksCreatedLastYear(): Unit = {
```

Comment:
It this being used?

Reply:
My question is other, why we have this method when we already have one totally equal who performs exactly the same.

The only difference between them is the name, please check the method "removeRoadWorksCreatedLastYear" to view what I'm talking about, and delete all modifications related to this method.

Reply:
Deleted this function since there is no use for this